fontforgeexe/pythonui.c

```diff
@@ -250,10 +253,13 @@ static void InsertSubMenus(PyObject *args,GMenuItem2 **mn, int is_cv) {
 	if ( mmn[j].ti.text==NULL ) {
 	    mmn[j].ti.text = submenuu;
 	    mmn[j].ti.fg = mmn[j].ti.bg = COLOR_DEFAULT;
+	    mmn[j].ti.mnemonic = mnemonic[i-5];
```

Comment:
i-5?

Reply:
`i` comes from Python argument index, which will at this point always be ≥ 5, which is why in the parent loop `i` = 5, because `cnt` == number of Python arguments.